source/_patterns/00-atoms/text/intro.twig

```diff
@@ -1 +1 @@
-<p class="o-intro">{{ text.intro | raw }}</p>
+<p class="o-intro text">{{ text.intro | raw }}</p>
```

Comment:
The text here is a bit repetitive.

Reply:
On second thought, this one should stay since it's an example.

Reply:
Oh, I think you're right. If the `.o-intro` is already wrapped in a `.text` class, then this one's not needed.